### PR TITLE
Webinar Flow Deployment

### PR DESCRIPTION
**Pull request by DiptoChakrabarty**
Created separate Docker  images  for react front-end and back-end and launched using compose file
with mongodb

### CODE REVIEW

File: webinar-flow/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: "3"
+services:
+  mongi:
```
Comment (L04DB4L4NC3R) [inline webinar-flow/docker-compose.yml]: Un authenticated database. Anyone can hit 27017 directly

File: webinar-flow/.env [hunk 2]
```diff
@@ -0,0 +1,9 @@
+admin=codechefAdmin
+
+secret=moveFastandBreakThings
+
+password=abc123
+
+mongoURL=mongodb+srv://pseudorandomGuy:mNyRGt8ZynzrPRf1@cluster0-axghx.mongodb.net/test?retryWrites=true&w=majority
```
Comment (L04DB4L4NC3R) [inline webinar-flow/.env]: Why????

File: webinar-flow/docker-compose.yml (hunk 1 above)
Comment (DiptoChakrabarty) [reply webinar-flow/docker-compose.yml]: @L04DB4L4NC3R  I deleted .gitignore by mistake thats why by mistake .env file got pushed.  I will delete this pull request and send a new pull request.

Comment (L04DB4L4NC3R) [reply webinar-flow/docker-compose.yml]: Change the password of the mongoDB cluster

Comment (DiptoChakrabarty) [reply webinar-flow/docker-compose.yml]: Yes I have told Subrah he said he will change it

**Pull request merged**